src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,180 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_ARRAY_CAPACITY = 16;
+    private static final int LOAD_FACTOR = 75;
```

Comment:
it may be confusing, let's change it to
```suggestion
    private static final double LOAD_FACTOR = 0.75;
```